WordPress/src/main/java/org/wordpress/android/models/Capability.java

```diff
@@ -0,0 +1,22 @@
+package org.wordpress.android.models;
+
+/**
+ * Used to decide what can the current user do in a particular blog
+ * A list of capabilities can be found in: https://codex.wordpress.org/Roles_and_Capabilities#Capabilities
+ */
+public enum Capability {
+    EDIT_USERS("edit_users"), // Check if user can change another user's role
+    LIST_USERS("list_users"), // Check if user can visit People page
+    PROMOTE_USERS("promote_users"); // Check if user can invite another user
+
+    private final String label;
+
+    Capability(String label) {
+        this.label = label;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
As said earlier, not an error per se so won't block the PR on this one.

Having said that, notice that `PostStatus` is not overriding the `.toString()` method and neither does `CommentStatus`.

It is true though that there are a few examples in the wpandroid codebase that do override it to return a semantically important value. I still think it's not appropriate and we should avoid continuing that practice. This fellow puts it better than me (when to override `.toString()`): http://stackoverflow.com/a/13291108